src/js/components/SharedComponents/navigation/NavigationComponent.jsx

```diff
@@ -32,21 +32,47 @@ export class Navbar extends React.Component {
         });
     }
 
-    render() {
-        let tabNames = {}
+    checkPermissions(){
+        if(!this.props.session.user.affiliations || this.props.session.user.affiliations.length == 0){
+            return false;
+        }
+        let aff = this.props.session.user.affiliations;
+        for(let i = 0; i < aff.length; i++){
+            if(aff[i].permission !== 'reader'){
```

Comment:
Is there a reason for using `!==` vs `!=` here? the "===" is strict and makes sure the data types are also the same, is that necessary here?